client/src/connection/studio/index.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { AxiosInstance } from "axios";
+
+function getStudioSessionRequest(connection: AxiosInstance) {
+  // Create a Date object representing the current date and time
+  const date = new Date();
+  // Get the time zone offset
```

Comment:
unneeded comment

Reply:
done in f5253ce014a5fec429456474725f717e0f2609a0